site/docs/BestPractice-ServingOverHTTP.md

```diff
@@ -44,7 +44,8 @@ A standard GraphQL POST request should use the `application/json` content type,
 ```js
 {
   "query": "...",
-  "variables": { variable1: value, ... }
+  "oprationName": "...",
```

Comment:
Yeah, maybe just add a sentence underneath the code snippet saying operationName and variables are optional.